frontend/src/api/ApiCaller.tsx

```diff
@@ -440,15 +440,6 @@ export class BackendAPICaller {
         return result.content
     }
 
-    static async getAreasMapMetadata(id: string): Promise<MapMetadata> {
-        const path: string = 'areas/' + id + '/map-metadata'
```

Comment:
Is the backend endpoint in use anymore? If not that can probably be removed as well

Reply:
Will investigate.

Reply:
Done.